tests/integration/test_events.py

```diff
@@ -1236,6 +1241,74 @@ def get_message(queue_url):
         # clean up
         self.cleanup(TEST_EVENT_BUS_NAME, rule_name, target_id, queue_url=queue_url)
 
+    @pytest.mark.parametrize(
+        "schedule_expression", ["rate(1 minute)", "rate(1 day)", "rate(1 hour)"]
+    )
+    @pytest.mark.aws_validated
+    def test_create_rule_with_one_unit_in_singular_should_succeed(
+        self, events_client, schedule_expression
+    ):
+        rule_name = self.create_rule_name()
+
+        # rule should be creatable with given expression
+        events_client.put_rule(Name=rule_name, ScheduleExpression=schedule_expression)
+
+        self.cleanup(rule_name=rule_name, events_client=events_client)
+
+    @pytest.mark.parametrize(
+        "schedule_expression", ["rate(1 minutes)", "rate(1 days)", "rate(1 hours)"]
+    )
+    @pytest.mark.xfail
+    def test_create_rule_with_one_unit_in_plural_should_fail(
+        self, events_client, schedule_expression
+    ):
+        rule_name = self.create_rule_name()
+
+        # rule should not be creatable with given expression
+        with pytest.raises(ClientError):
+            events_client.put_rule(Name=rule_name, ScheduleExpression=schedule_expression)
+
+    @pytest.mark.xfail
+    def test_verify_rule_event_content(self, events_client, logs_client):
+        log_group_name = "/aws/events/testLogGroup-{}".format(short_uid())
+        logs_client.create_log_group(logGroupName=log_group_name)
+
+        log_groups = logs_client.describe_log_groups(logGroupNamePrefix=log_group_name)
+        assert len(log_groups["logGroups"]) == 1
+        log_group = log_groups["logGroups"][0]
+
+        log_group_arn = log_group["arn"]
+
+        rule_name = self.create_rule_name()
+        events_client.put_rule(Name=rule_name, ScheduleExpression="rate(1 minute)")
+
+        target_id = "testRuleId-{}".format(short_uid())
+        events_client.put_targets(Rule=rule_name, Targets=[{"Id": target_id, "Arn": log_group_arn}])
+
+        # wait one minute, as then the rule will trigger and send the event to the log group
+        time.sleep(61)
```

Comment:
To address your concern about the sleep here:
The issue with this is that if for some reason it took more than 60s (which technically is fine since it's AFAIK not guaranteed that after the put_targets call, the timer is immediately running) this test would fail even though the behavior is valid.

In this case we'd add a polling/waiting mechanism to repeatedly try to assert what you're waiting for (e.g. logs).

There are mutliple ways to do this. For some actions there are waiter integrated into boto3 via `client.get_waiter(...)`, for most generic waiting situations though you should use one of the util functions in `localstack.utils.sync`